osh/hub/scan/notify.py

```diff
@@ -197,10 +197,8 @@ def generate_general_text(self, display_states=True):
     Fix later -- defect is true positive, but fix is postponed to next release
     Not a bug -- issue is false positive, so you are waiving it
             """
-        message += """
```

Comment:
Previous footer included a newline at the beginning.

Reply:
So we should prepend a newline to `settings.NOTIFICATION_EMAIL_FOOTER` to have the footer delimited by a blank line in both the cases.

Reply:
I already prepend a newline at line 140 to workaround that. How many newlines should be there before the footer?

Reply:
Line 140 creates a different message does not it?  There should be a blank line between `Not a bug -- issue is false positive, so you are waiving it` and `This is a test footer.`, too.

Reply:
Does not the line 203 add a newline? Shall that be removed?

Reply:
Nope.  I think you need a similar code change as above:
```
    if settings.NOTIFICATION_EMAIL_FOOTER:
        message += "\n"
        message += settings.NOTIFICATION_EMAIL_FOOTER
```

Otherwise you are replacing a blank line (`\n\n`) with a new line (`\n`).  And if `display_state` was `False`, there would be no delimiter at all with the proposed change.

Reply:
Ok, sorry for the confusion, but let me clarify again. Do you consider the message that starts with `You can find documentation of OpenScanHub` footer?

Reply:
Yes but I am not sure how it is related to the problems outlined above.  I think the footer set by `settings.NOTIFICATION_EMAIL_FOOTER` should be delimited from the notification body.